content/en/getting-started/configuration.md

```diff
@@ -394,6 +418,18 @@ See [Content Management](/content-management/urls/#permalinks).
 
 (`bool`) Whether to pluralize automatic list titles. Applicable to section pages. Default is `true`.
 
+###### printI18nWarnings
+
+(`bool`) Print missing translations. Default is `false`.
```

Comment:
```suggestion
(`bool`) Whether to print missing translations to the console. Default is `false`.
```

Reply:
my suggestion - see global comment:
  (`bool`) Whether to log WARNINGs for each missing translation. Default is `false`.